sub_cal/sub_cal/src/main/java/model/Operation.java

```diff
@@ -0,0 +1,72 @@
+package model;
+
+import java.math.BigDecimal;
+import java.util.Stack;
+import java.util.StringTokenizer;
+public class Operation {
+    private Stack<Double> numberStack = new Stack<>();
+    private Stack<String> operStack = new Stack<>();
+    private static final int CONVERTTONEGATIVE = -1;
+
+    public String operate(String expression) {
+
+        StringTokenizer st = new StringTokenizer(expression);
+
+        classifyOperatorAndNumber(st);
+
+        return calculate();
+    }
+    private void classifyOperatorAndNumber(StringTokenizer st){
+
+        while(st.hasMoreTokens()){
+            String word = st.nextToken();
+            Operator operator = null;
+            if(!(Operator.getOperator(word).isEmpty())){
+                operator = Operator.getOperator(word).get();
+            }
+
+            switch (operator != null ? operator : operator.NULL){
```

Comment:
`getOperator`에서 NULL을 반환하고 있는데 굳이 null인지 검사할 필요가 있을까요?
Optional을 반환하는 것이 아니라 항상 Operator을 반환할 수 있도록 하면될 것 같아요

```suggestion
            Operator operator = Operator.getOperator(word);
            switch (operator){
```

Reply:
말씀해주신대로 수정하였습니다 .. !